docs/topics/autothrottle.rst

```diff
@@ -47,12 +47,19 @@ effect, but there are some important differences:
 
 AutoThrottle doesn't have these issues.
 
-Disabling throttling on a downloader slot
-=========================================
+.. reqmeta:: dont_throttle
 
-It is possible to disable AutoThrottle for a specific download slot at run time
-by setting its ``throttle`` attribute to ``False``, e.g. using
-:setting:`DOWNLOAD_SLOTS`.
+Disabling the throttling of a request
+=====================================
+
+To disable AutoThrottle for a specific request, set the ``dont_throttle``
+request metadata key to ``True``:
+
+.. code-block:: python
+
+    from scrapy import Request
+
+    Request("https://example.com", meta={"dont_throttle": True})
 
 Note, however, that AutoThrottle still determines the starting delay of every
```

Comment:
There is another caveat: dont_throttle disables the adjustment of the slot delay, but not the delay itself. So, for example, if AutoThrottle was enabled for first N requests to a website, and then there is a request with dont_throttle=True, then the last request is still going to use the delay learned previously.

It seems the `dont_throttle` name could be confusing in this case: we do throttle the request. It seems the flag works more like `autothrottle_dont_adjust_delay` (is there a shorter name?)

Reply:
Would `bypass_autothrottle` (or `skip_`/`ignore_`) work? Otherwise, I’m OK with `autothrottle_dont_adjust_delay`.